src/aws_sdk_dynamodbstore.rs

```diff
@@ -474,44 +592,72 @@ impl super::Backend for Backend {
             .unwrap_or(HashMap::new()))
     }
 
+    #[tracing::instrument(skip_all, fields(key, consumed_wcu, otel.status_code, error.msg, otel.span_kind = "client"))]
+
     async fn z_add<'a, 'b, K: Key<'a>, V: Into<Arg<'b>> + Send>(&self, key: K, value: V, score: f64) -> Result<()> {
         let v = value.into();
         self.zh_add(key, &v, &v, score).await
```

Comment:
This will introduce double spans, as both `z_add` and `zh_add` are instrumented. iirc this is why the Rusoto one has the `zh_add_impl`.

Reply:
That explains a lot about the structure difference I noticed, I'll refactor the structures to match.

Reply:
Yeah, an explanatory comment wouldn't have hurt. It was probably past-me that forgot to put one in.